lib/msal-browser/FAQ.md

```diff
@@ -212,6 +213,11 @@ loginPopup().then((response) => {
 })
 ```
 
+## Is SSO enabled when I have multiple MSAL instances running?
+
+Yes. We try to SSO by default unless the cache policies say otherwise.
```

Comment:
This does not have enough context and is very open to interpretation imo - I would either expand on this or remove it, our acquireToken docs should cover this topic in more detail.

Reply:
Removed. The FAQ initially was added as a guidance for `config` param and now I agree that it does not have enough context to qualify for an FAQ.